app/pages/reporter/index.tsx

```diff
@@ -79,7 +78,7 @@ export default class Reporter extends Component<Props> {
     return (
       <DefaultLayout showSubheader session={session} title="My Operators">
         <Row>
```

Comment:
I'm guessing you removed the 8-column width on the `Col` below in order to center it on the page? This will center it while maintaining the original page content width. I'd recommend sticking with that because once content gets too spread too widely apart it can be hard to associate related items (in this case, both the steps and the table row columns are quite far apart).

```suggestion
        <Row className="justify-content-md-center">
          <Col md={10} lg={8}>
```

Reply:
<img width="1440" alt="Screen Shot 2021-04-22 at 7 21 41 AM" src="https://user-images.githubusercontent.com/5522075/115733119-723f0f80-a33d-11eb-967d-cf550412434b.png">

Reply:
I'd recommend sticking with the scope of this feature (progress bar) and maintain the existing layout, as there are other appropriate venues to rethink page layout.